src/components/about/svg/about.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="150" height="250" viewBox="0 0 197 300" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M103.322 31.8608C101.068 27.1208 100.112 21.7946 101.907 16.5932C107.081 1.60642 127.451 10.6543 129.519 19.4694C131.587 28.2846 125.845 47.7069 120.127 47.6548C117.846 47.634 113.665 45.0732 109.788 41.0511L107.18 62.3501L90.0424 57.7677L103.322 31.8608Z" fill="#B28B67"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M127.02 13.3569C126.691 13.0178 126.002 12.3043 125.634 12.014C125.547 11.9446 124.99 11.2688 124.969 11.27C124.4 11.3065 122.714 11.5521 122.434 12.0789C122.402 12.1161 122.372 12.1507 122.343 12.1805C122.025 12.5123 121.721 12.8586 121.425 13.2121C120.776 13.9882 120.172 14.8193 119.586 15.6474C118.445 17.2601 117.38 18.9378 116.36 20.6319C115.321 22.3565 114.357 24.1246 113.343 25.8641C111.884 28.3671 110.237 30.749 108.783 33.2522C107.855 34.8506 107.128 36.5245 106.774 38.3304C106.769 38.3521 106.765 38.3733 106.761 38.3955C106.759 38.4045 106.758 38.4136 106.756 38.4227C106.704 38.7008 106.66 38.982 106.626 39.2667C106.598 39.5103 106.569 39.7548 106.543 39.9998C107.566 38.8502 108.467 37.5232 109.38 36.2789C109.469 36.1571 109.556 36.0336 109.642 35.9094C109.892 35.6743 110.144 35.4422 110.399 35.2116C112.002 33.7578 113.608 32.3116 115.173 30.8136C117.988 28.1193 120.804 25.3522 123.91 23.0175C124.828 22.3281 125.758 21.6563 126.668 20.9574C127.187 20.5598 127.697 20.151 128.198 19.73C128.556 19.4289 130.027 18.4391 129.828 17.5548C129.726 17.1024 127.875 14.2367 127.02 13.3569Z" fill="#2C2C2C"/>
```

Comment:
This line is coming up as a different colour and not sure what could be causing it..